### DISCUSSION TITLE
Where can I find maintainers?

### DISCUSSION THREAD

**Discussion by Devanshukoli** [EddieHubCommunity/LinkFree#9724] [Q&A] [answered]
- So, I was wondering about, is there any kind of list of maintainers or something? 
- if there, where can I find them?

- So, why am I asking this question?
- Because when I created <chore> type of **PR**, then I thought I should let the maintainers know about this. so I was wondering to whom to tag? I thought I should tag **eddie** but then again. I find it hard to do.

**Comment by kumarsonsoff3** [comment, marked as answer]
> So, I was wondering about, is there any kind of list of maintainers or something?

Yep, you'll find that at https://www.biodrop.io/maintainers

> so I was wondering to whom to tag? I thought I should tag eddie but then again. I find it hard to do.

You need not to tag any maintainer specifically, it will only create extra noise for everyone because every maintainer gets notifications of all the activity on the project. 

Also, Our maintainers are working as volunteers from different timezones, so it might take us a while to review your PR.
